src/server/shell/basic_idle_handler.h

```diff
@@ -42,6 +44,30 @@ namespace shell
 {
 class DisplayConfigurationController;
 
+class BasicIdleHandlerObserverMultiplexer: public ObserverMultiplexer<IdleHandlerObserver>
```

Comment:
I started wondering why this was at namespace scope, and then realised that it is used by `Dimmer` and `PowerModeSetter` (which actually only need the base interface `IdleHandlerObserver`). Perhaps not worth changing but...

Reply:
Worth changing IMO :+1: